src/pages/getting-started/livechat-apps/index.mdx

```diff
@@ -46,6 +42,127 @@ See a 4-minute tutorial explaining how to create a simple app in the Developer C
   allowfullscreen
 ></iframe>
 
+
+## Step-by-step guide
+
+If you want to know the whole creation process in detail, follow the instructions we’ve prepared. We'll use an app named **Your Guide** as our example.
+
+### **Step 1 Getting started**
+
+Make sure you are signed in to the [Developer Console](https://developers.livechat.com/console/). Your LiveChat credentials should allow you to be logged in automatically. If not, you can [sign up here](https://accounts.livechat.com/signup).
+
+Now, we go to the **Apps** tab and click the **Create new app +** button. Give your app a name and click the **Continue** button. You can always change the application's name later.
+
+💡 **Need a helping hand?** Right after signing in to the Developer Console, you can launch the **App wizard**. It will guide you through the whole app creation process.
+
+<img
+  alt="Creating LiveChat apps step 1, App wizard"
+  src="/images/getting-started/livechat-apps/LiveChat-apps-step1.png"
+  width="1515px"
+  height="819px"
+/>
+
+### **Step 2 Adding Agent App Widgets block**
+
+You can configure your app using Building blocks which are components that form your app. Depending on which building block you add to your application, it will work differently. If you want to know how each building block can enrich your app, click the **Read docs** redirect and learn more about them. 
```

Comment:
```suggestion
You can configure your app using Building blocks, which we can describe as components that form your app. Depending on which building block you add to your application, it will work differently. If you want to know how each building block can enrich your app, click the **Read docs** redirect and learn more about them. 
```